src/CMakeLists.txt

```diff
@@ -9,6 +8,42 @@ include_directories("md/inc")
 include_directories("classlibnative/bcltype")
 include_directories("classlibnative/cryptography")
 include_directories("classlibnative/inc")
+include_directories("${GENERATED_INCLUDE_DIR}")
+
+if(WIN32)
+    add_custom_command(
+      COMMENT "Generating Eventing Files"
+      OUTPUT ${GENERATED_INCLUDE_DIR}/ClrEtwAll.h
+      OUTPUT ${GENERATED_INCLUDE_DIR}/clretwallmain.h
+      OUTPUT ${GENERATED_INCLUDE_DIR}/etmdummy.h
+      COMMAND ${MC} -h ${GENERATED_INCLUDE_DIR} -r ${GENERATED_INCLUDE_DIR} -b -co -um -p FireEtw "${VM_DIR}/ClrEtwAll.man"
+      COMMAND ${PYTHON} -B -Wall -Werror "${CLR_DIR}/src/scripts/genWinEtw.py" --man "${VM_DIR}/ClrEtwAll.man" --exc "${VM_DIR}/ClrEtwAllMeta.lst" --eventheader "${GENERATED_INCLUDE_DIR}/ClrEtwAll.h" --macroheader "${GENERATED_INCLUDE_DIR}/clretwallmain.h" --dummy "${GENERATED_INCLUDE_DIR}/etmdummy.h"
+      DEPENDS "${CLR_DIR}/src/scripts/genWinEtw.py" "${VM_DIR}/ClrEtwAll.man" "${VM_DIR}/ClrEtwAllMeta.lst"
+    )
+else()
+    add_custom_command(
```

Comment:
Please unify these for Unix / win32. They differ only in the python script, so you can just create a CMake variable containing the script path in the if / else and use it in the add_custom_command that you move out.